.circleci/config.yml

```diff
@@ -156,7 +156,7 @@ jobs:
       - bootstrap_repository_command
       - run:
           name: Check if packages are ready to be released
-          command: npm run release:prepare-packages -- --verbose --compile-only
+          command: yarn release:prepare-packages -- --verbose --compile-only
```

Comment:
Please revert this change. Yarn and npm do not share a `npm login` session. Hence, we use `npm run` to release stuff instead of `yarn run`.